xbmc/playlists/PlayListM3U.cpp

```diff
@@ -178,7 +178,7 @@ bool CPlayListM3U::Load(const std::string& strFileName)
           newItem->SetProperty("item_start", iStartOffset);
           newItem->SetEndOffset(iEndOffset);
           // Prevent load message from file and override offset set here
-          newItem->GetMusicInfoTag()->SetLoaded();
+          // newItem->GetMusicInfoTag()->SetLoaded(); - This prevents information being loaded for multi-song files later
```

Comment:
so what behaviour does this change?

We cant just comment a line and leave it in. What was your reasoning that gave you pause to not just delete the line?

Reply:
Only so that I can comment why I'd removed it. Will remove it.
As per the comment - if you mark the `MusicInfoTag` as loaded there (which is isn't really) then no information gets added later on.

Reply:
I'd question why (what) this prevents being loaded later (background loader maybe?).  However, I'd also suggest that CUE sheets are an obsolete method of loading songs, date from ~2000 and apart from a few niche cases (mka files - which I intend to address) are pretty much pointless these days.

I know we like to support legacy stuff as far as possible but I'd love to drop cue sheet support rather than extend or fix any issues with it.

Reply:
I don't disagree but feel we should fix it and then depreciate it - then people can make the choice of uisng the last versoin that works or upgrading.

If i remember rightly it's because this:

```  if ((!pItem->HasMusicInfoTag() || !pItem->GetMusicInfoTag()->Loaded()) && pItem->IsAudio())```
(line 173 in MusicInfoLoader.cpp)

is false and the song info never getsl loaded.